pagopa/io-app
# fix: [IABT-1364] Add accessibility fixes to the checkout page

## Short description
This PR is going to fix various accessibility issues in the checkout page.

<details>
<summary>Payment method preview</summary>
<img width="1414" alt="Screenshot 2022-03-31 at 11 10 25" src="https://user-images.githubusercontent.com/5963683/161020658-06e03e7f-b9e6-4a3a-ad38-632b549fdfec.png">
<img width="1416" alt="Screenshot 2022-03-31 at 11 11 25" src="https://user-images.githubusercontent.com/5963683/161020648-f6e72c62-c92b-4fa8-98d2-c3a657be74ef.png">
</details>

<details>
<summary>PSP preview</summary>
<img width="1417" alt="Screenshot 2022-03-31 at 11 25 10" src="https://user-images.githubusercontent.com/5963683/161023167-4bfeebbe-821a-479d-b3f1-cf4a2b2b3f05.png">
<img width="1420" alt="Screenshot 2022-03-31 at 11 26 51" src="https://user-images.githubusercontent.com/5963683/161023633-7b8aa188-0154-4411-9f93-e710311b7c20.png">
</details>

## List of changes proposed in this pull request
- Added the second line of informations to the accessibility label of each payment method
- Fixed a typo in the PSP accessibility label
- The PSP box accessibility label now shows a `tap to edit` only when there's actually that action

## How to test
Using the accessibility inspector in the checkout page:
- The payment method box should read the second line of informations too (name, surname and expiration date for a generic payment method)
- The PSP box should not read "Provided by" two times and the `tap to edit` text should be present only if needed
